ci-operator/step-registry/telco5g/cnf/tests/telco5g-cnf-tests-commands.sh

```diff
@@ -21,6 +21,16 @@ sriov "FPGA Programmable Acceleration Card N3000 for Networking"
 # TESTNAME
 dpdk "Client should be able to forward packets"
 
+# SKIPTEST
+# bz### https://issues.redhat.com/browse/OCPBUGS-11046
+# TESTNAME
+tuningcni "sysctl allowlist update should start a pod with custom sysctl only after adding sysctl to allowlist"
+
+# SKIPTEST
+# bz### https://issues.redhat.com/browse/OCPBUGS-10927
+# TESTNAME
+xt_u32 "Validate the module is enabled and works Should create an iptables rule inside a pod that has the module enabled"
```

Comment:
If I understood correctly from Carlos in this issue, it shouldn't run on rhel9 only. So it should be skipped for 13 and 14, but 11 and 12 can continue run it.

Reply:
> Should I create an additional function for 4.14?

Yeah